### DISCUSSION TITLE
404 for just one file?

### DISCUSSION THREAD

**Discussion by VAggrippino** [community/community#22943] [Other Feature Feedback, Questions, & Ideas]
<p>Why would GitHub Pages fail to serve a single file when it’s serving the rest of the page without error?</p>
<p>I’m hosting an <a href="https://vaggrippino.github.io/CVRemix/" rel="nofollow noopener">online version of my Curriculum Vitae</a> with GitHub Pages. It works fine, but there’s one 404 error in the console. It’s for a &lt;video&gt; element with multiple &lt;source&gt; elements. The first choice (mp4) doesn’t load, so one of the alternates (webm) does.</p>
<table><tbody>
<tr>
<td>404 at:</td>
<td><a href="https://vaggrippino.github.io/CVRemix/Coverr/MP4/Love-coding.mp4" target="_blank" rel="nofollow noopener">https://vaggrippino.github.io/CVRemix/Coverr/MP4/Love-coding.mp4</a></td>
</tr>
<tr>
<td>no 404 at:</td>
<td><a href="https://vaggrippino.github.io/CVRemix/Coverr/WEBM/Love-Coding.webm" target="_blank" rel="nofollow noopener">https://vaggrippino.github.io/CVRemix/Coverr/WEBM/Love-Coding.webm</a></td>
</tr>
</tbody></table>
<p>The repo is at <a href="https://github.com/VAggrippino/CVRemix" rel="nofollow noopener">https://github.com/VAggrippino/CVRemix</a></p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/vaggrippino">@vaggrippino</a>,</p>
<p>You have a typo which is causing this problem. Your mp4 file is located at <a href="https://vaggrippino.github.io/CVRemix/Coverr/MP4/Love-Coding.mp4" rel="nofollow noopener">https://vaggrippino.github.io/CVRemix/Coverr/MP4/Love-Coding.mp4</a> with both capital “L” and capital “C” whereas your link has only a capital “L”.</p>
<p>Thanks!</p>

**Comment by VAggrippino** [comment]
<p>:grimacing: That figures… I checked and double-checked and still didn’t see that. It just goes to show that it always helps to have a second set of eyes. Thank you!</p>
